### PR TITLE
Support Rails 5.1

### PR DESCRIPTION
**Pull request by maschwenk**
Rails 5.1 release has been officially cut http://weblog.rubyonrails.org/2017/4/27/Rails-5-1-final/

### CODE REVIEW

File: lib/has_scope/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module HasScope
-  VERSION = "0.7.0"
+  VERSION = '0.8.0'.freeze
```
Comment (rafaelfranca) [inline lib/has_scope/version.rb]: Can you avoid bumping the gem and the adding a changelog header? I don't think we are planning to release a new minor release

**Comment by rafaelfranca** [comment]
Thank you

**Comment by varyonic** [comment]
Does that mean we are waiting on @lucasmazza to bump version and `rake release`?

**Comment by rafaelfranca** [comment]
I can do the release. I was just waiting the week to come to avoid working in the weekend 😄

**Pull request merged**